### DISCUSSION TITLE
Volume not multiplied error*

### DISCUSSION THREAD

**Discussion by EYErbil** [deepmodeling/dpdata#681] [Q&A] [answered]
Hello, When running e_f with dpdata output.out -i cp2k/output -o deepmd/npy -O data , and using the recommended input files, I get the error:

`` 
--- You are parsing data using package Cp2kData ---                                                                     Atom names are fake chemical symbols as you set in cp2k input.                                                          Virial Parsing using cp2kdata as plug in for dpdata                                                                     was not multiplied by volume before cp2kdata v0.6.4                                                                     please check the cp2kdata version and the virial.npy                                                                                                                                                                                            --- You are parsing data using package Cp2kData --- 
 ``
my cp2kdata is updated, can anyone help?

**Comment by robinzyb** [comment, marked as answer]
this is a warning. if the cp2kdata is updated, you can ignore it
